src/locales/locale-en.json

```diff
@@ -684,6 +685,32 @@
     "SUCCESS": "The request was successfully sent. Have a look at your Emails!",
     "TITLE": "Password reset"
   },
+  "PLACE_TYPE_NAMES": {
+    "Unspecified": "Unspecified",
+    "Store": "Store",
+    "Sharing Point": "Sharing Point",
+    "Meeting Place": "Meeting Place",
+    "Restaurant": "Restaurant",
+    "Market": "Market"
+  },
+  "PLACE_TYPE_STATUS": {
+    "ACTIVE": "Active",
+    "ARCHIVED": "Archived"
+  },
+  "PLACE_TYPES": {
+    "ADD": "Add new place type",
+    "NAME": "Name",
+    "STATUS": "Status",
+    "STANDARD_NAME_HEADING": "Standard names",
+    "STANDARD_NAME_HINT": "Name will be available in other languages",
+    "CUSTOM_NAME_HINT": "Name will be used exactly as you write it and not translated",
+    "CUSTOM_NAME_USE": "Use custom name {name}",
+    "CUSTOM_NAME_PROMPT": "Type a custom name, or choose a standard name",
+    "MANAGE_TYPES": "Manage types...",
+    "SHOW_ARCHIVED": "Show archived",
```

Comment:
Many of these translations are duplicate from activity types, I have been tending to re-use things in this case, even if the i18n keypath is a bit nonsensical... and then imagining if we can migrate keypaths properly one day (did we manage this before?)

Reply:
They are duplicated, yeah.. I think it's not too bad, these small messages are fast to translate. Transifex has some hints when a string was already translated, you can just click and it fills the previous one.